cyvcf2/cyvcf2.pyx

```diff
@@ -1494,6 +1494,8 @@ cdef class Variant(object):
             if self._gt_types == NULL:
                 self._gt_phased = <int *>stdlib.malloc(sizeof(int) * self.vcf.n_samples)
                 ngts = bcf_get_genotypes(self.vcf.hdr, self.b, &self._gt_types, &ndst)
+                if ngts < 0:
+                    raise Exception("error parsing genotypes")
```

Comment:
can you add a hint here that they may simply be absent, e.g.: `"error parsing genotypes; they may be absent from this record"` .